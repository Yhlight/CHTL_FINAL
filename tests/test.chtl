div {
    style {
        // Simple operations
        width: 10px + 20px; // 30px
        height: 50% - 20%; // 30%
        font-size: 2em * 3; // 6em
        margin: 100px / 4; // 25px
        padding: 10px % 3px; // 1px
        border-width: 2px ** 3px; // 8px

        // Operator precedence
        left: 10px + 5px * 2; // 20px

        // Mixed units (should fallback to calc)
        top: 10px + 5%;

        // Non-expression values
        color: red;
        content: "hello";

        // Negative numbers
        bottom: -10px + 5px; // -5px

        // Unit-less multiplication
        font-size: 1.2em * 2; // 2.4em

        // Division with same units
        line-height: 20px / 10px; // 2
    }
}
