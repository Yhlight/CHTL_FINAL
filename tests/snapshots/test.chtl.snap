<div style="border-width: 8px; bottom: -5px; color: red; content: hello; font-size: 2.4em; height: 30%; left: 20px; line-height: 2px; margin: 25px; padding: 1px; top: 15px; width: 30px; " />
